launch-jupyter.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Please remove the files that are not in the recipes directories.

Reply:
Inside the recipe directory that will use them.

Reply:
Done